rest/app.py

```diff
@@ -61,14 +70,20 @@ def get_user_tasks(user_id):
         year = period['year']
         month = period.get('month', None)
         day = period.get('day', None)
+        logger.info('Get tasks for user_id "{user_id}" for year={year}, '

```

Comment:
Please , use more a siple way for format without name. 
'{} {} {}'.format(a, b, c)